builder.py

```diff
@@ -0,0 +1,134 @@
+class Control:
```

Comment:
please take a look at this file containing implementation of builder
[main.txt](https://github.com/asidonlylips/func_git/files/3116492/main.txt)